bip-0155.mediawiki

```diff
@@ -101,7 +101,7 @@ The list of reserved network IDs is as follows:
 | <code>0x03</code>
 | <code>TORV2</code>
 | 10
-| Tor v2 hidden service address
+| Tor v2 hidden service address (**Tor v2 is no longer operational**; clients **MUST NOT** gossip or relay Tor v2 addresses and **MUST** ignore them on receive).
```

Comment:
The MediaWiki format uses single quotes to format text as follows: ```''italic''```, ```'''bold'''```, ```'''''bold & italic'''''```.

Reply:
Good to know, thanks.